src/migration/v1-addon/steps/create-file-path-maps/app/index.ts

```diff
@@ -1,12 +1,10 @@
 import type { FilePathMap, Options } from '../../../../../types/index.js';
 import { mapComponentClasses } from './map-component-classes.js';
-import { mapComponentStylesheets } from './map-component-stylesheets.js';
 import { mapComponentTemplates } from './map-component-templates.js';
 
 export function mapAppFolder(options: Options): FilePathMap {
   return new Map([
     ...mapComponentClasses(options),
-    ...mapComponentStylesheets(options),
```

Comment:
I had added `mapComponentStylesheets()` because a production project had sometimes included `styles.js` in the `app` folder. As far as I know, `ember-css-modules` never required this for podded addons.